code/modules/mining/machine_redemption.dm

```diff
@@ -160,6 +160,8 @@
 
 /obj/machinery/mineral/ore_redemption/default_unfasten_wrench(mob/user, obj/item/I)
 	. = ..()
+	if(!.)
```

Comment:
I think you need to check for `SUCCESSFUL_UNFASTEN` instead. If `..()` returns `FAILED_UNFASTEN` it'll still pass.

Reply:
```suggestion
	if(. != SUCCESSFUL_UNFASTEN)
```